lib/Doctrine/Common/Cache/CacheProvider.php

```diff
@@ -175,6 +176,20 @@ private function getNamespacedId(string $id) : string
     {
         $namespaceVersion = $this->getNamespaceVersion();
 
+        return $this->getNamespacedIdForVersion($id, $namespaceVersion);
+    }
+
+    /**
+     * Return the passed id prefixed with the namespace value and suffixed
+     * with the passed namespace version number
+     *
+     * @param string $id               The id to namespace.
```

Comment:
We can drop these `@param` and `@return` since the information there is already provided by the type declaration